plugins/source/aws/resources/services/support/trusted_advisor_checks.go

```diff
@@ -0,0 +1,43 @@
+package support
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/support"
+	"github.com/aws/aws-sdk-go-v2/service/support/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+func TrustedAdvisorChecks() *schema.Table {
+	return &schema.Table{
+		Name:        "aws_support_trusted_advisor_checks",
+		Description: `https://docs.aws.amazon.com/awssupport/latest/APIReference/API_DescribeTrustedAdvisorChecks.html`,
+		Resolver:    fetchTrustedAdvisorChecks,
+		Transform:   transformers.TransformWithStruct(&types.TrustedAdvisorCheckDescription{}, transformers.WithPrimaryKeys("Id")),
+		Multiplex:   client.ServiceAccountRegionsLanguageCodeMultiplex("support", []string{"en", "fr", "de", "id", "it", "ja", "ko", "pt_BR", "es", "zh", "zh_TW"}),
```

Comment:
Could extract the language list as a global []string here (unexported) so that it can be reused in the test as well. Otherwise the list being long could cause testing issues in the future (eg. test cases being forgotten for a newly added lang code)

Reply:
I think the test will fail it that case with a missing call (you need to mock each specific language code), but I'll do that anyways